include/avt_vimba_camera/avt_vimba_api.h

```diff
@@ -100,6 +100,33 @@ class AvtVimbaApi {
     return "Unsupported error code passed.";
   }
 
+  std::string interfaceToString(VmbInterfaceType interfaceType) {
+    switch ( interfaceType ) {
+      case VmbInterfaceFirewire: return "FireWire";
+        break;
+      case VmbInterfaceEthernet: return "GigE";
+        break;
+      case VmbInterfaceUsb: return "USB";
+        break;
+      default: return "Unknown";
+    }
+  }
+
+  std::string accessModeToString(VmbAccessModeType modeType) {
+    std::string s;
+    if (modeType & VmbAccessModeFull)
+      s = std::string("Read and write access");
```

Comment:
How about `return \"Read and write access\"` so it matches `interfaceToString` function?

Reply:
Sounds good! this function was just moved from another file, I'll improve it though.